judo-ui-react/src/main/resources/actor/src/layout/Header/HeaderContent/Profile/index.tsx.hbs

```diff
@@ -0,0 +1,220 @@
+                  maxWidth: 250
+                }
+              } }
+            >
+              <ClickAwayListener onClickAway={handleClose}>
+                <Card>
+                  <CardContent sx={ { px: 2.5, pt: 3 } }>
+                    <Grid container justifyContent="space-between" alignItems="center">
+                      <Grid item>
+                        <Stack direction="row" spacing={1.25} alignItems="center">
+                          <Avatar alt="profile user" src={heroProps.imgSrc} sx={ { width: 32, height: 32 } } />
+                          <Stack>
+                            <Typography variant="h6">{heroProps.preferredUsername}</Typography>
+                            <Typography variant="body2" color="textSecondary">
+                              {t('applications.{{ application.actor.simpleName }}', { defaultValue: '{{ application.actor.simpleName }}' })}
+                            </Typography>
+                          </Stack>
+                        </Stack>
+                      </Grid>
+                      {{# if application.authentication }}
+                      <Grid item>
+                        <Tooltip title="Logout">
+                          <IconButton size="large" sx={ { color: 'text.primary' } } onClick={doLogout}>
+                            <MdiIcon path="logout" />
+                          </IconButton>
+                        </Tooltip>
+                      </Grid>
+                      {{/ if }}
+                    </Grid>
+                  </CardContent>
+
+                  <Box sx={ { borderBottom: 1, borderColor: 'divider' } }>
+                    <Tabs variant="fullWidth" value={value} onChange={handleChange} aria-label="profile tabs" sx={ { height: '20px', alignItems: 'center' } }>
+                      <Tab
+                        sx={ {
+                          display: 'flex',
+                          flexDirection: 'row',
+                          justifyContent: 'center',
+                          alignItems: 'center',
+                          textTransform: 'capitalize'
+                        } }
+                        icon={<MdiIcon path="account-outline" sx={ { marginBottom: 0, marginRight: '10px' } } />}
+                        label="Profile"
+                        {...a11yProps(0)}
+                      />
+                      {/*<Tab*/}
+                      {/*  sx={ {*/}
+                      {/*    display: 'flex',*/}
+                      {/*    flexDirection: 'row',*/}
+                      {/*    justifyContent: 'center',*/}
+                      {/*    alignItems: 'center',*/}
+                      {/*    textTransform: 'capitalize',*/}
+                      {/*  } }*/}
+                      {/*  icon={<MdiIcon path="cog-outline" sx={ { marginBottom: 0, marginRight: '10px' } } />}*/}
+                      {/*  label="Setting"*/}
+                      {/*  {...a11yProps(1)}*/}
+                      {/*/>*/}
```

Comment:
Will we need this later?

Reply:
I'm keeping it in yes. It's more cumbersome to dig it out later.